Social Login and Org Switcher (#6161)
Kong/docs.konghq.com: app/konnect/org-management/org-switcher.md

---
title: Org Switcher
content_type: how-to
---

The org switcher allows a user with multiple {{site.konnect_short_name}} accounts to seamlessly switch between organizations they belong to. You can navigate to the org switcher by clicking on **Your Org Name** > **Back to Org Switcher**.

From the org switcher, you can:
- View a list of {{site.konnect_short_name}} organizations you have access to
- Login to a different {{site.konnect_short_name}} organization
- Create a new {{site.konnect_short_name}} organization

## Email Matching

> The org switcher allows users to switch between {{site.konnect_short_name}} accounts with the same email address as the user currently logged in. Each organization that contains an account with the same email address will be listed in the org switcher.

Reviewer: ```suggestion
The org switcher uses the email of the logged in user to locate all organizations where this email is attached to a {{site.konnect_short_name}}. 
```

- Be invited to an organization - Organizations that the user is invited to will automatically show up in the org switcher.

{:.note}
> **Note**: It is possible to end up with more than one org switcher identities if the user does not choose to “Link Accounts” when logging into {{site.konnect_short_name}} for the very first time on different social credentials. Organizations will only be associated with one primary account. It is not currently possible to re-link accounts. {{site.konnect_short_name}} recommends “Linking Accounts” when using multiple social identities.

## Switch Organizations

Users can switch organizations while logged in by clicking on the **Org Name** > **Back to Org Switcher**.
The user can login to each organization listed in the org switcher by clicking on the Select  button.

If the organization is configured to be single-sign-on only, then the user will be directed to the configured identity provider to re-authenticate into {{site.konnect_short_name}}.

{:.note}
> **Note**: You cannot be logged in to more than one {{site.konnect_short_name}} organization at the same time. You must return to the org switcher in order to switch to another organization.

> ## Create a New Organization
> 
> The user can create a new organization by clicking on the **+ Create** button.

Reviewer: ```suggestion
```
